main.js

```diff
@@ -0,0 +1,122 @@
+    image: ' assets/images/img3.svg',
+    languages: ['HTML', 'CSS', 'Ruby on rails', 'JavaScript'],
+    technologies: ['HTML', 'CSS', 'Ruby on rails', 'JavaScript', 'GitHub'],
+    seeLive: 'https://bludidi.github.io/Portfolio-Desktop-version-/',
+    seeSource: 'https://github.com/Bludidi/Portfolio-Desktop-version-',
+  },
+
+  {
+    name: 'Uber Navigation',
+    subtitle: 'UBER',
+    backEnd: 'Leader Developer',
+    year: 2018,
+    shortDescription: 'A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.',
+    description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the relea",
+    image: 'assets/images/img4.svg',
+    languages: ['HTML', 'CSS', 'JavaScript'],
+    technologies: ['HTML', 'CSS', 'Ruby on rails', 'JavaScript', 'GitHub'],
+    seeLive: 'https://bludidi.github.io/Portfolio-Desktop-version-/',
+    seeSource: 'https://github.com/Bludidi/Portfolio-Desktop-version-',
+  },
+];
+
+const projectCards = projects.map((project) => `
+<div id="card1" class="cards">
+            <div id="card_img" class="card left_align"><img class="card_img" src="${project.image}" alt="project card"></div>
+              <div id="txt_img" class="img-txt1"><h2>${project.name}</h2>
+                <ul class="flex-cards">
+                  <li class="all-list first">${project.subtitle}</li>
+                  <li class="all-list second">${project.backEnd}</li>
+                  <li class="all-list second">${project.year}</li>
+                </ul>
+                <p class="p3">${project.shortDescription}</p>
+                <ul class="flex-button tech_lang">
+                ${project.languages.map((tech) => `<li class="button">${tech}</li>`).join('')}
+                </ul>
+                <br>
+                <input class="btns see_btn" type="button" value="See Project">
+            </div>
+        </div>
```

Comment:
- [ ] Kindly follow  design for  mobile versions from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colours, images, text, margins). Please try to adjust the alignment based on the Figma design.
The Figma design Look likes: 
![Capt3](https://user-images.githubusercontent.com/93598189/190614692-80cd020b-3002-4afb-a4c6-681fbd124dac.PNG)

Your design Look Likes : 
![Capt4](https://user-images.githubusercontent.com/93598189/190614880-a4a84a8e-f9d1-4776-9365-fadcee6766a4.PNG)